### PR TITLE
typo in a .css filename

### PR DESCRIPTION
**Pull request by bartocc**
probably a copy/paste error ;-)

### CODE REVIEW

**Comment by Gaurav0** [comment]
I'm not sure. I think the agreed way is to add the styles to the addon tree, not the app tree. @rwjblue ?

In that case, the error is in the previous line.

File: guides/writing-addons/intro-tutorial.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ We can do this by creating stylesheets in the `app/styles/` directory instead. T
 Let's create `app/styles/our-addon-name.css` and add a rule to it:
 
 ```css
-/* addon/styles/our-addon-name.css */
+/* app/styles/our-addon-name.css */
```
Comment (mansona) [inline guides/writing-addons/intro-tutorial.md]:
```suggestion
/* addon/styles/addon.css */
```

**Comment by mansona** [comment]
@bartocc any chance you can update this PR so that we can get it merged?

**Comment by bartocc** [comment]
@mansona @Gaurav0 the edit in this PR is for a paragraph named `CSS stylesheets that require importing from addons`

To let the dev decide wether or not he/she wants to import the addon's CSS in the consuming app, we need to place the CSS in the addon's `app` folder.

If we place it in the `addon` folder, it will be **automatically** imported

Could you guys re-read the paragraph [CSS stylesheets that require importing from addons](https://cli.emberjs.com/release/writing-addons/intro-tutorial/#cssstylesheetsthatrequireimportingfromaddons)  and make sure that we are not misunderstanding each other 🙏?

**Comment by mansona** [comment]
@bartocc interesting 🤔 I didn't see that because, as you suspected, I missed which context we were actually talking about here. I did cross-reference the file but I didn't notice that there were 2 sections about this 😂 

So this brings up 2 problems: 

- the preceding section is also wrong 
- I'm not 100% sure this suggestion would actually work unless you have a css preprocessor included in your build pipeline 🤔 

Does anyone know this one way or the other?

**Comment by bartocc** [comment]
@mansona I am closing this PR in favor of #109 

My suggested change had already been implemented in #101 , but with some typos

**Pull request closed without merging**